src/poetry/console/commands/source/add.py

```diff
@@ -28,20 +29,31 @@ class SourceAddCommand(Command):
             (
                 "Set this source as the default (disable PyPI). A "
                 "default source will also be the fallback source if "
-                "you add other sources."
+                "you add other sources. (DEPRECATED)"
```

Comment:
```suggestion
                "you add other sources. (<warning>Deprecated</warning>)"
                " Use --priority instead."
```

Same for `secondary`.

Reply:
Changed into `<warning>Deprecated, use --priority</warning>`